runners/spark/pom.xml

```diff
@@ -196,6 +196,11 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
+      <groupId>commons-lang</groupId>
```

Comment:
I needed it for `WordUtils#capitalize` in `SparkNativePipelineVisitor.NativeTransform#replaceFnString`